base/server/share/webapps/pki/js/pki-ui.js

```diff
@@ -184,7 +184,12 @@ var Dialog = Backbone.View.extend({
     saveField: function(input) {
         var self = this;
         var name = input.attr("name");
-        var value = input.val();
+        var value;
+        if (name == "status") {
+            value = self.entry.nextStates[input[0].selectedIndex - 1];
+        } else {
+            value = input.val();
+        }
```

Comment:
Is `status` a TPS token-specific attribute? In that case you might want to add a `saveField()` in the `TokenDialog` class instead to perform TPS token-specific operation.

Reply:
Yes it maps to the `TokenStatus` class in Java-land - I have created the new `saveField()` method.